### ISSUE TITLE
Not getting better results after training for several days?

### ISSUE THREAD

**Issue by alan-ai-learner** [frankplus/meena-chatbot#4]
Hi @frankplus, as you got the better results then might be  the problem is from my  side so, i used this [dataset](http://opus.nlpl.eu/download.php?f=OpenSubtitles/v2018/mono/OpenSubtitles.raw.en.gz) , so please take a look into this dataset and let me know. Can you suggest me a dataset for the english conversation

Thanks

**Comment by frankplus** [comment]
Hi thanks for the issue,
maybe try the [tokenized version](https://opus.nlpl.eu/download.php?f=OpenSubtitles/v2018/mono/OpenSubtitles.en.gz) instead otherwise you would have to tokenize yourself.

**Comment by alan-ai-learner** [comment]
okay thanks for the response, so should i use the dataset link provided by you directly, in the code.?

**Comment by frankplus** [comment]
Yes, and if you can improve the dataset preprocessing, I think it could be a bit better.

**Comment by alan-ai-learner** [comment]
@dataset is too huge so I'm unable to analyze the data and don't know what part  I should look into that, Any way if you got any suggestions then i wanna know, if not will look into it. 
Thanks
